pkg/ebpf/tracer.go

```diff
@@ -17,6 +18,14 @@ import (
 	bpflib "github.com/iovisor/gobpf/elf"
 )
 
+var (
+	probeExpvar *expvar.Map
+)
+
+func init() {
+	probeExpvar = expvar.NewMap("systemprobe")
```

Comment:
Any reason we can't just do this in `NewTracer`?

Reply:
That's how expvar works, it has to be in `init()` to setup, or all our tests would failed because it can only be declared once(when the module is loaded).